common/app/common/JsMinifier.scala

```diff
@@ -33,18 +32,14 @@ object JsMinifier {
     options.setWarningLevel(DiagnosticGroups.DEPRECATED_ANNOTATIONS, CheckLevel.WARNING)
     options.setWarningLevel(DiagnosticGroups.DEBUGGER_STATEMENT_PRESENT, CheckLevel.WARNING)
     options.setWarningLevel(DiagnosticGroups.CHECK_REGEXP, CheckLevel.WARNING)
-    options.setWarningLevel(DiagnosticGroups.UNNECESSARY_CASTS, CheckLevel.WARNING)
```

Comment:
Removed in https://github.com/google/closure-compiler/commit/9165f761984c7a7fb72b54a26098a984cfd8455e